go.mod

```diff
@@ -1,10 +1,10 @@
 module go.mongodb.org/atlas-sdk/v20231115008
 
-go 1.22.1
+go 1.20
```

Comment:
why not only rollback this particular change?

Reply:
Updated description. We want to rollback whole change to make build pass.